config/examples/max_allowed_days.yml

```diff
@@ -0,0 +1,12 @@
+max_allowed_days: &default
+  account_inactivity: 365
+
+development:
```

Comment:
We usually do not include `development` and `test` for the OpenShift version of the configs. No need to change anything but let's just remember that without `/config/docker/max_allowed_days.yml` and `openshift/system/config/max_allowed_days.yml` this one will be the default settings applying to OpenShift deployment, and not https://github.com/3scale/porta/blob/d89bcb2ce8335c203afb7c9302b30b21c7e76f81/app/lib/max_allowed_days_loader.rb#L7

Reply:
This is for examples, what you copy for your development environment

Agree for `config/docker` and `openshift/system/config`

Reply:
@guicassolato @hallelujah So nothing to do for `config/docker` and `openshift/system/config`, right? that would just mean that it is a disabled feature 😄 Otherwise, what is the desired config for those?

Reply:
Yes correct :)

Reply:
You mean "disabled features" because of the rolling update to be added in the second part? Because `/config/max_allowed_days.yml` will be copied and loaded.